src/style.css

```diff
@@ -0,0 +1,42 @@
+body {
+  margin: 0;
+}
+
+.important,
+.interesting,
+.surprising,
+#header,
+#footer {
+  width: 300px;
+  height: 100px;
+  margin: auto;
+}
+
+.important:hover,
+.interesting:hover,
+.surprising:hover,
+#header:hover,
+#footer:hover {
+  height: 200px;
+  margin: auto;
```

Comment:
why you repeat margin? it doesnt change on hover

Reply:
Deleted unnecessary margin